docs/CSBR.md

```diff
@@ -503,19 +503,16 @@ If the CA decides that the revocation will have an unreasonable impact on its cu
+Except for cases that fall under Section 4.9.1.1, all incidents that lead the CA to believe that the certificate private key is compromised or is being used for Suspect Code, CAs SHALL revoke the Code Signing Certificate in accordance with and within the following maximum timeframes. Nothing herein prohibits a CA from revoking a Code Signing Certificate prior to these timeframes.
 
-1.  The CA MUST contact the software publisher within one (1) business day after the CA is made aware of the incident.
-2.  The CA MUST determine the volume of relying parties that are impacted (e.g., based on OCSP logs) within 72 hours after being made aware of the incident.
-3.  The CA MUST request the software publisher send an acknowledgement to the CA within 72 hours of receipt of the request.
-  a.  If the publisher responds within 72 hours, the CA and publisher MUST determine a "reasonable date" to revoke the certificate based on discussions with the CA.
-  b.  If CA does not receive a response, the CA must notify the publisher that the CA will revoke in 7 days if no further response is received.
-    i.  If the publisher responds within 7 days, the CA and the publisher will determine a "reasonable date" to revoke the certificate based on discussion with the CA.
-    ii.  If no response is received after 7 days, the CA must revoke the certificate except if the CA has documented proof (e.g., OCSP logs) that the revocation will cause significant impact to the general public.
+1.  The CA MUST contact the Subscriber within 24 hours after the CA is made aware of the incident.
+2.  The CA MUST request the Subscriber send an acknowledgement to the CA within 72 hours of receipt of the request.
+   a.  If the Subscriber responds within 72 hours, the CA and Subscriber MAY determine a "reasonable date" to revoke the certificate based on discussions with the CA. The revocation date MUST NOT be more than 7 days after the incident was discovered.
+   b.  If CA does not receive a response, then the CA MUST revoke the certificate within 24 hours from the end of the response period.
 
-A CA revoking a Certificate because the Certificate was associated with signed Suspect Code or other fraudulent or illegal conduct SHOULD provide all relevant information and risk indicators to other CAs or industry groups. The CA SHOULD indicate whether its investigation found that the Suspect Code was a false positive or an inadvertent signing.
+A CA revoking a Certificate because the Certificate was associated with signed Suspect Code or other fraudulent or illegal conduct SHOULD provide all relevant information and risk indicators to other CAs or industry groups. The CA SHOULD indicate whether its investigation found that the Suspect Code was a false positive or an inadvertent signing. The CA SHOULD contact the Application Software Suppliers within 24 hours after the CA is made aware of an incident involving use in Suspect Code.

```

Comment:
I realize the text "its investigation found that the Suspect Code was a false positive" wasn't modified or added as part of this ballot but I'm wondering if we can improve that language in the scope of this ballot, mainly because I don't know what it means.

I *think* it's saying that CAs SHOULD report Suspect Code that turned out to be a false positive to other CAs. If it indeed carries that intent, then I'm struggling to understand why that practice would be beneficial to the ecosystem.

Reply:
Actually this is a SHOULD in case of actual positives, but I see the confusion. Would it be clearer if we switched places for the last 2 sentences?

Reply:
I guess I'm still not seeing how switching the order of the final two sentences will help. There's seemingly a SHOULD-level obligation for the CA to indicate whether the Suspect Code was a false positive, i.e. the Suspect Code was actually legitimate and no action should have been taken. If the conclusion of the investigation is that there was no Suspect Code signed, then where is the CA "indicating" this?